mechanics/TOC-REVIEW.schedule

```diff
@@ -54,3 +54,12 @@
 2022-03-03T08:30:00-07:00 | Eventing WG
 2022-03-10T08:30:00-07:00 | Networking WG
 2022-03-17T08:30:00-07:00 | DUX WG
+2022-03-31T08:30:00-08:00 | Security WG
+2022-04-07T08:30:00-07:00 | Productivity WG
+2022-04-14T08:30:00-07:00 | Client WG
+2022-04-21T08:30:00-07:00 | Serving API WG
+2022-04-28T08:30:00-07:00 | Scaling WG
+2022-05-05T08:30:00-07:00 | Operations WG
+2022-05-12T08:30:00-07:00 | Eventing WG
+2022-05-19T08:30:00-07:00 | Networking WG
```

Comment:
This is the week of Kubecon.

Adjust this, or #979 to do the same thing because I didn't look at Slack this morning